script/bootstrap

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
 set -e
-go get -u github.com/golang/lint/golint
```

Comment:
Any reason to stop using golint?

Yay for replacing `go get` with `go mod`!

Reply:
When i was trying to get **golint** package after getting the package it seems that we had an issue and when i used go mod instead of it got all the dependencies and issue was resolved so that's why removed golint